diff -urN aa/gnome-python-extras-2.9.3/configure.ac gnome-python-extras-2.9.3/configure.ac
--- aa/gnome-python-extras-2.9.3/configure.ac	2005-01-25 22:04:35.000000000 +0100
+++ gnome-python-extras-2.9.3/configure.ac	2005-02-10 23:23:35.237264560 +0100
@@ -222,7 +222,7 @@
 if test -n "$export_dynamic"; then
   GTKMOZEMBED_LIBS=`echo $GTKMOZEMBED_LIBS | sed -e "s/$export_dynamic//"`
 fi
-MOZILLA_HOME="`$PKG_CONFIG --variable=libdir $mozpackage-gtkmozembed`"
+MOZILLA_HOME="`$PKG_CONFIG --variable=libdir $mozpackage-plugin`"
 AC_SUBST(MOZILLA_HOME)
 
 
