diff -urN gnome-python-extras-2.25.3/gtkmozembed/Makefile.am gnome-python-extras-2.25.3.new/gtkmozembed/Makefile.am
--- gnome-python-extras-2.25.3/gtkmozembed/Makefile.am	2008-12-12 02:06:29.000000000 +0100
+++ gnome-python-extras-2.25.3.new/gtkmozembed/Makefile.am	2009-04-26 00:42:54.423462020 +0200
@@ -1,7 +1,7 @@
 # require automake-1.5
 AUTOMAKE_OPTIONS = 1.5
 
-INCLUDES = $(PYTHON_INCLUDES) $(PYGTK_CFLAGS) $(GTKMOZEMBED_CFLAGS)
+INCLUDES = $(PYTHON_INCLUDES) $(PYGTK_CFLAGS) $(GTKMOZEMBED_CFLAGS) -I/usr/include/xulrunner/gtkembedmoz
 
 defsdir = $(datadir)/pygtk/2.0/defs
 defs_DATA = gtkmozembed.defs
